analyzers/SonarAnalyzer.sln

```diff
@@ -57,7 +57,9 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "SonarAnalyzer.Shared", "src
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "GitHub", "..\.github\GitHub.shproj", "{532AC70E-90CB-48AD-86F4-16767C22314D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SonarAnalyzer.ShimLayer.CodeGeneration", "src\SonarAnalyzer.ShimLayer.CodeGeneration\SonarAnalyzer.ShimLayer.CodeGeneration.csproj", "{684F0AD7-BBC9-40C8-9E54-D9C2B57560D9}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SonarAnalyzer.ShimLayer.CodeGeneration", "src\SonarAnalyzer.ShimLayer.CodeGeneration\SonarAnalyzer.ShimLayer.CodeGeneration.csproj", "{684F0AD7-BBC9-40C8-9E54-D9C2B57560D9}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "its", "its\its.shproj", "{AF787E26-7AB1-4B08-AF05-07CB0ECDE4E5}"
```

Comment:
If we don't come up with good idea for `<something>` from the other comment, let's just make it `ITs` and leave it in `Common Solution Items`
```suggestion
Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "ITs", "its\its.shproj", "{AF787E26-7AB1-4B08-AF05-07CB0ECDE4E5}"
```

Reply:
I kept the spelling of the folder, based on your comment:
> ITs is actually a real folder in the project structure, just as Tests.

Reply:
I'd still prefer `ITs` as it's easier to read. `.github` is also lowercase on the filesystem